pyproject.toml

```diff
@@ -1,7 +1,7 @@
 [tool.black]
 # bump the dev dependency version when updating this
 # also bump black version in the pre-commit config
-required-version = "24.3.0"
+required-version = "24.8.0"
```

Comment:
It looks like there are some new `--enable-unstable-feature` options (per line 10 in this file)
```
is_simple_lookup_for_doublestar_expression
docstring_check_for_newline
remove_redundant_guard_parens
parens_for_long_if_clauses_in_case_block
```
Do any of these seem beneficial to include? Didn't look too hard, not sure if any of these are documented here https://black.readthedocs.io/en/stable/usage_and_configuration/the_basics.html#enable-unstable-feature

Reply:
Ah, here we go https://black.readthedocs.io/en/stable/the_black_code_style/future_style.html

Reply:
According to the future_style doc, these unstable features have moved into the --preview mode that we have enabled, so let's remove them from `enable-unstable-feature` list on line 11
```
"hex_codes_in_unicode_sequences",
"unify_docstring_detection",
"no_normalize_fmt_skip_whitespace",
"typed_params_trailing_comma",
```

Reply:
> It looks like there are some new --enable-unstable-feature options (per line 10 in this file)

These features are already included in preview mode (they probably were when I first added enable-unstable-feature). No action required